File: app/src/main/java/com/college/app/utils/ObjectBox.java
package com.college.app.utils;

import android.content.Context;

import com.college.app.MyObjectBox;

import io.objectbox.BoxStore;

public class ObjectBox {

    private static BoxStore boxStore;

    public static void init(Context context) {
        boxStore = MyObjectBox.builder()
                .androidContext(context.getApplicationContext())
                .build();
    }

    public static BoxStore getBoxStore() {
        return boxStore;
    }
}

File: app/src/main/java/com/college/app/utils/ObjectBox.kt
package com.college.app.utils

import android.content.Context
import com.college.app.MyObjectBox
import io.objectbox.BoxStore

object ObjectBox {
    var boxStore: BoxStore? = null
        private set

    fun init(context: Context) {
        boxStore = MyObjectBox.builder()
            .androidContext(context.applicationContext)
            .build()
    }
}